.github/workflows/docs.yml

```diff
@@ -4,13 +4,16 @@ on:
     branches:
       - 'master'
     paths:
-      - 'docs/**'
+      - 'DSharpPlus.Docs/**'
   release:
     types:
       - published
+  workflow_dispatch:
+  
 env:
   DOTNET_NOLOGO: true
   DOTNET_CLI_TELEMETRY_OPTOUT: true
+  DOTNET_INSTALL_DIR: "~/.dotnet"
```

Comment:
Why are we changing the install dir?

Reply:
my runner was dying because of dir being /usr/share/dotnet